### PR TITLE
[MRG] Functions order + git presentation

### PR DESCRIPTION
**Pull request by glemaitre**
Change the order of the functions and a comment such that people used the solution before the module creation.

### CODE REVIEW

File: Day_2_Software_engineering_best_practices/05_documentation_git.html [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+<p style="margin-bottom: 3cm;"></p>
+
+.affiliations[
+  ![:scale 80%](../img/logoUPSayPlusCDS_990.png)
+]
+
+---
+
+# What `git` is used for?
```
Comment (jorisvandenbossche) [inline Day_2_Software_engineering_best_practices/05_documentation_git.html]: maybe use here version control isntead of git?

File: Day_2_Software_engineering_best_practices/05_documentation_git.html [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+
+# Concepts of `git`
+
+### Common commands
+
+* `$ git init`: initialize a `git` repository.
+* `$ git add file`: start tracking changes for a file.
+* `$ git rm file`: stop tracking changes for a file.
+* `$ git status`: get information regarding the status of the repository.
+* `$ git commit`: get a snapshot of the changes of the files tracked.
```
Comment (jorisvandenbossche) [inline Day_2_Software_engineering_best_practices/05_documentation_git.html]: get -> take ?

File: Day_2_Software_engineering_best_practices/05_documentation_git.html [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+```
+
+### 6. Check the commit history
+
+We can now check the history.
+
+```bash
+$ git log
+```
+
```
Comment (jorisvandenbossche) [inline Day_2_Software_engineering_best_practices/05_documentation_git.html:221]: do we still want to do the next step here for adding the python files and commiting them?

File: Day_2_Software_engineering_best_practices/05_documentation_git.html [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+count: false
+# What `git` is used for?
+
+* Keep track of history
+* Safe development of new features
+* Swap between versions
+* **Enable collaborative coding, writing, ...**
+
+---
+
```
Comment (jorisvandenbossche) [inline Day_2_Software_engineering_best_practices/05_documentation_git.html]: maybe put here a slide that introduces git -> one system for version control + link

**Comment by glemaitre** [comment]
@glemaitre Remember to put some links for extra material

**Pull request merged**